src/sentry/static/sentry/app/components/events/rrwebIntegration.jsx

```diff
@@ -0,0 +1,111 @@
+import PropTypes from 'prop-types';
```

Comment:
Ca nyou write as typescript?

Reply:
@dcramer 

> oh wait
> nvm expert now
> dethroning evan

And you won't even write this component in typescript!

Reply:
https://github.com/getsentry/sentry-rrweb/blob/master/src/index.ts pro